GSA-TTS/FAC
# Display Search Timing

# Display Search Timing

Issue: https://github.com/GSA-TTS/FAC/issues/3372

Part of some small changes that will resolve the above issue.

## Changes:
Display the time that a search took next to the results count. 

Also, remove `.lighthouseci` from the gitignore. That should've come along in a previous PR. 

## How to test:
1. Switch to this branch and run normally.
2. Navigate to and make a search. Verify that the time the search took displays above the results table.
3. Verify the same for the advanced search screen.

## Screenshots:
Basic:
![Screen Shot 2024-05-24 at 11 26 43 AM](https://github.com/GSA-TTS/FAC/assets/91098850/ba0cfb42-260c-469e-9230-4f224a03922e)


Advanced:
![Screen Shot 2024-05-24 at 11 21 29 AM](https://github.com/GSA-TTS/FAC/assets/91098850/26a69767-623e-4f15-8239-ed2369b8aaae)





## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
